Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -564,9 +566,12 @@ - (void)viewWillAppear:(BOOL)animated
     // We need to `beginLongLivedReadTransaction` before we update our
     // mapping in order to jump to the most recent commit.
     [self.uiDatabaseConnection beginLongLivedReadTransaction];
+    self.messageMappings = [[YapDatabaseViewMappings alloc] initWithGroups:@[ self.thread.uniqueId ]
+                                                                      view:TSMessageDatabaseViewExtensionName];
```

Comment:
This statement - creating a new instance of the mappings - is the part of this PR I'm least sure of.  It might not be necessary.